code/__DEFINES/dcs/signals/signals_mob/signals_mob_living.dm

```diff
@@ -176,3 +176,6 @@
 
 /// From /obj/structure/geyser/attackby() : (obj/structure/geyser/geyser)
 #define COMSIG_LIVING_DISCOVERED_GEYSER "living_discovered_geyser"
+
+/// From /datum/ai/behavior/climb_tree/perform() : (mob/living/basic/bear_pawn)
```

Comment:
not fond of the name `bear_pawn` when we've just genericized it to fit any mob, kindly change it to something like `(mob/living/basic/basic_pawn)` or something